WORKSPACE

```diff
@@ -135,3 +135,10 @@ git_repository(
 load("@io_bazel_rules_d//d:d.bzl", "d_repositories")
 
 d_repositories()
+
+# For structure tests
+git_repository(
```

Comment:
Can you wait for @nkubala's PR to land?  IDK how close he is.

Reply:
sure. i can work with him to land his PR in.

Reply:
Meanwhile do review other changes...